### ISSUE TITLE
Author Ships

### ISSUE THREAD

**Issue by TheGiraffe3** [endless-sky/endless-sky#10224]
### Problem Description

A while back, @Quantumshark, @roadrunner56, and @tibetiroka were added as Devs. However, they did not get their own author ships. 

### Related Issue Links

Haven’t found any. 

### Desired Solution

That Quantumshark, roadrunner56, and tibetiroka get author (or person) ships. 

### Alternative Approaches

Just not give them author ships and they can continue to go unrecognized? Not a great choice.

**Comment by Saugia** [comment]
For what it's worth, being a Developer doesn't necessarily equate to owning an Author Ship. Some of us devs do not have person ships to begin with, and some with person ships in the game have never been a Dev.

Generally through the past Author ships follow a combination of a lot of content in the game, be it tons of artwork reworks, a whole faction, or major reworks to storylines, etc., with the nomination from several individuals. Of course, things like the Reviewer ships, or Subsidurial, are more exceptions (as one refers to people as a group, and the other is a reference).

**Comment by tibetiroka** [comment]
Related: #5296

**Comment by TheGiraffe3** [comment]
True, but @Saugia you should definitely get an author ship. After all you made a whole species.

**Comment by Saugia** [comment]
Yes; we've been going back and forth with Peter on discord about adjustments and what ends up in the game and referencing that.

**Comment by RisingLeaf** [comment]
That is for others to decide, although I don't think my contributions are on the same level as other people with an author ship.
